sympy/polys/numberfields/__init__.py

```diff
@@ -0,0 +1,52 @@
+"""Computational algebraic field theory. """
+
+__all__ = [
+    'minpoly', 'minimal_polynomial', 'primitive_element',
+
+    'field_isomorphism',
+
+    'to_number_field', 'isolate',
+
+    'nilradical_mod_p', 'round_two',
+
+    'prime_valuation', 'prime_decomp',
+]
+
+from .basis import nilradical_mod_p, round_two
+
+from .primes import prime_valuation, prime_decomp
+
+# Before `sympy.polys.numberfields` was a package, it was a module, containing
+# what is now defined in the `minpoly`, `isomorphism` and `numbers` modules.
+# To maintain backwards compatibility, we have to import everything that was
+# available in the old `sympy.polys.numberfields` module.
+# The lists below may contain some (harmless) redundancies between them, since
+# each was automatically generated by scanning the module.
```

Comment:
I think that one probably should remove, at least, the obvious ones not originating from those modules (like S, Rational, Add, Dummy, etc). 

Also, you will need to mark these to avoid the test errors, I think with `# noqa: F401`.

Reply:
Okay, I agree that omitting `Rational`, `Add`, etc. is cleaner and is probably the right thing. My intention was to cater to those users who may have -- against all good coding principles! -- imported such things _from this module_. But, I think we could hardly be faulted for failing to support that kind of behavior!

May need some clarification on the `# noqa: F401` thing, but will cross that bridge when I come to it.....